plugins/catalog-backend-module-ldap/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@backstage/plugin-catalog-backend-module-ldap",
-  "version": "0.2.1",
-  "main": "src/index.ts",
+  "version": "0.2.2-0.18",
```

Comment:
The version doesn't need to be updated. If the PR has a changeset file, this will be updated by the bot with the next release.